modal/functions.py

```diff
@@ -1070,6 +1082,8 @@ def _hydrate_metadata(self, metadata: Optional[Message]):
         self._is_method = metadata.is_method
         self._use_function_id = metadata.use_function_id
         self._use_method_name = metadata.use_method_name
+        self._class_parameter_format = metadata.class_parameter_format
+        self._class_parameter_schema = list(metadata.class_parameter_schema)
```

Comment:
j/w but what type is `metadata.class_parameter_schema` before you cast it to a list? I couldn't figure it out.

Reply:
It's some `protobuf.RepeatedYadaYada`... I made the cast to satisfy some weird mypy check